### ISSUE TITLE
Not importing highlights (calibre plugin)

### ISSUE THREAD

**Issue by luisneto** [degiz/kobo2calibre#8]
I've installed the calibre plugin, but it does not import highlights.

**Steps:**
- Connect Kobo to the computer;
- Launch calibre;
- Select a book and press the Kobo2Calibre button;
- A popup is shown saying how many highlights there are in the copy stored on the Kobo;
- Click OK to proceed;

**Result:**
A pop is shown saying "Inserted 0 highlights into calibre"

I've tried this with various books.
I am using a Kobo Libra 2, a MacBook and calibre 6.10.

**Comment by degiz** [comment]
Hey @luisneto 

Are you using [calibre-kobo-driver](https://github.com/jgoguen/calibre-kobo-driver) to convert the books to kepub?

**Comment by luisneto** [comment]
I have the `KoboTouchExtended` plugin installed but the "Send books as kepubs" option is disabled.
I also have the `KePub Output Plugin` plugin but I am not using it.

It's not a requirement for your plugin to work, is it?

**Comment by degiz** [comment]
> It's not a requirement for your plugin to work, is it?

I only tested the plugin on the books converted using the `KoboTouchExtended` (from https://github.com/jgoguen/calibre-kobo-driver), I've mentioned that in the Readme.

My plugin relies on the conversion from epub > kepub (using a regexp, which I found in the code), hence I can "restore" the location of the highlights, and put it back to calibre.

I don't know if that would be possible to rely on the internal Kobo's conversion, unfortunattely.

**Comment by luisneto** [comment]
> I've mentioned that in the Readme.

You're right, there's a sentence there, but in my opinion, that sentence does not make it clear what the requirements are.

I'm not looking to do any conversions. Only to transfer the highlights.

How is it supposed to work for books purchased in Kobo, and therefore exist as KEPUB in the e-reader and each matches with an EPUB in Calibre? There is no conversion taking place here.

> I don't know if that would be possible to rely on the internal Kobo's conversion, unfortunattely.

Which Kobo internal conversion?

**Comment by AndrewHyman** [comment]
Just installed successfully! This is an amazing plugin. Highlights are working perfectly, but the annotation's notes are not importing and visible in Calibre. Is this currently supported?

**Comment by degiz** [comment]
Hey @AndrewHyman , that's not currently supported, but should be quite easy to add.
